### DISCUSSION TITLE
edit nginx conf

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#4104] [General] [answered]
Hi All!

simple qst: How can I edit and persist nginx container data. I need to add a new redirect to the nginx conf
Thanks

**Comment by weslambert** [comment, marked as answer]
For the time being, you can copy `/opt/so/saltstack/default/salt/nginx/etc/nginx.conf` to `/opt/so/saltstack/local/salt/nginx/etc/nginx.conf`, and make your changes there.
